model/parameterSheet.cc

```diff
@@ -49,31 +50,28 @@ namespace
   };
 }
 
-void ParameterSheet::populateItemvector() {
-  itemvector.clear();	
-  minsky().canvas.selection.recursiveDo(&GroupItems::items,
+void ParameterSheet::populateItemVector() {
+  itemVector.clear();	
+  minsky().canvas.model->recursiveDo(&GroupItems::items,
                                         [&](Items&, Items::iterator i) {                                 
-                                          itemvector.emplace_back(*i);
+                                          itemVector.emplace_back(*i);
```

Comment:
You could test for item being a parameter before placing this in the itemVector.